cmd/kubeadm/app/constants/constants.go

```diff
@@ -81,8 +81,8 @@ const (
 	MinimumAddressesInServiceSubnet = 10
 
 	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
-	// Default behaviour is "never expire" == 0
-	DefaultTokenDuration = 0
+	// Default behaviour is 2 hours
+	DefaultTokenDuration = 2 * time.Hour
```

Comment:
As you said this have possibility to break existing users, @luxas Do we have any rollout plan to avoid this?